internal/core/textlayout.rs

```diff
@@ -422,7 +447,7 @@ fn test_elision() {
                 .collect::<Vec<char>>()
         })
         .collect::<String>();
-    debug_assert_eq!(rendered_text, "This is a lon…")
+    debug_assert_eq!(rendered_text, "This is a lo…")
```

Comment:
the test sets `max_width: 13. * 10.` and uses a fixed width font with 10px wide glyphs so it should be 12 glyphs of text plus the elide glyph.